### ISSUE TITLE
JRE, JDK, Oracle or OpenJDK?

### ISSUE THREAD

**Issue by otzoran** [pires/docker-jre#12]
the README says 

> Lean JRE 8 Docker image, based on Alpine Linux 3.7 and **OpenJDK** 8 

the URL under "Current software" sends to **Oracle's JDK** (not JRE)(not OpenJDK)
the Dockerfile ultimately installs **openjdk8-jre**. A bit confusing...

regards

**Comment by pires** [comment]
Indeed. But I can't find the release notes for the OpenJDK releases :( can you help?

**Comment by otzoran** [comment]
found this [link](http://openjdk.java.net/projects/jdk8u).
